ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -561,24 +563,13 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
         return dismissPresentingViewController(animated: true)
     }
 
-    func updateCurrentTokens(_ tokens: DrawerTokens) {
-        self.tokens = tokens
-    }
+    public typealias TokenSetType = DrawerTokenSet
+    public typealias TokenSetKeyType = DrawerTokenSet.Tokens
+    public var tokenSet: DrawerTokenSet = .init()
```

Comment:
I don't quite have the time to dig into this to figure it out myself right now, but does this still have the inheritance issue I was hitting when trying to update the PopupMenu?

Reply:
It shouldn't, because token lookups will now be indexed by their unique `TokenSetKey` rather than by control type (which can be poisoned by inheritance).